packages/dd-trace/src/runtime_metrics/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+let runtimeMetrics
+
+const noop = runtimeMetrics = {
+  start: () => {},
+  stop: () => {},
+  track: () => {},
+  boolean: () => {},
+  histogram: () => {},
+  count: () => {},
+  gauge: () => {},
+  increment: () => {},
+  decrement: () => {}
+}
+
+module.exports = {
+  start: (config) => {
+    if (!config?.runtimeMetrics) return
+
+    runtimeMetrics = require('./runtime_metrics')
+    runtimeMetrics.start(config)
+  },
+
+  stop: () => {
+    if (runtimeMetrics === noop) return
+
+    runtimeMetrics.stop()
+    runtimeMetrics = noop
+  },
+
+  track: (...args) => runtimeMetrics.track(...args),
```

Comment:
Rather than barreling functions through like this, maybe it's better to do a prototype switch.

Reply:
Made the change 👍

Reply:
Any remaining concerns?